roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -20,8 +20,8 @@ class DockerImageAvailability(DockerHostMixin, OpenShiftCheck):
     # to look for images available remotely without actually pulling them.
 
     # command for checking if remote registries have an image, without docker pull
-    skopeo_command = "{proxyvars} timeout 10 skopeo inspect --tls-verify={tls} {creds} docker://{image}"
-    skopeo_example_command = "skopeo inspect [--tls-verify=false] [--creds=<user>:<pass>] docker://<registry>/<image>"
+    skopeo_command = "{proxyvars} timeout 10 skopeo --debug inspect --tls-verify={tls} {creds} docker://{image}"
+    skopeo_example_command = "skopeo --debug inspect [--tls-verify=false] [--creds=<user>:<pass>] docker://<registry>/<image>"
```

Comment:
You can probably take `--debug` off this line because it is just printed on error.  This will also resolve the pylint error.

Reply:
Fixed this.